services/data-info/src/data_info/routes/domain/stats.clj

```diff
@@ -59,8 +59,8 @@
   {:file (describe FileStatInfo "File info")})
 
 (s/defschema PathsMap
-  {(describe String "The IRDOS data item's path")
-   (describe (s/either FileStatInfo DirStatInfo) "The data item's info")})
+  {(describe s/Keyword "The IRDOS data item's path")
+   (describe (s/conditional #(contains? % :file-size) FileStatInfo :else DirStatInfo) "The data item's info")})
```

Comment:
This wouldn't document correctly, but it's not in the branch that actually produces the documentation; that one uses either as before.